pyproject.toml

```
@@ -0,0 +1,76 @@
+[project]
+name = "parastell"
+description = "Parametric 3-D CAD modeling toolset for stellarator fusion devices"
+version = "0.0.2"
+
+readme = "README.md"
+requires-python = ">=3.11.0"
+maintainers = [
+	{name = "Connor Moreno", email = "camoreno@wisc.edu"},
+	{name = "Paul Wilson", email = "paul.wilson@wisc.edu"}
+]
+
+authors = [
+	{name = "Connor Moreno"},
+	{name = "Edgar Pflug"},
+	{name = "Paul Wilson"},
+	{name = "Eitan Weinstein"},
+	{name = "Enrique Miralles-Dolz"},
+	{name = "Joshua Smandych"},
+	{name = "Paul Romano"},
+	{name = "Bowen Quan"},
+	{name = "Jonathan Shimwell"}
+]
+
+keywords = ["stellarator", "fusion", "reactor", "parametric", "cad", "model", "dagmc", "geometry"]
+license = {file="LICENSE.md"}
+classifiers = [
+       "Development Status :: 3 - Alpha",
+       "Intended Audience :: Science/Research",
+       "Intended Audience :: Developers",
+       "License :: OSI Approved :: MIT License",
+       "Natural Language :: English",
+       "Operating System :: OS Independent",
+       "Topic :: Scientific/Engineering :: Physics",
+       "Topic :: Scientific/Engineering",
+       "Topic :: Scientific/Engineering :: Human Machine Interfaces",
+       "Programming Language :: Python :: 3 :: Only",
+       "Programming Language :: Python :: 3.11",
+]
+
+dependencies = [
+	"numpy~=1.26.4",
+	"scipy",
+	"cadquery",
+	"pymoab~=5.5.1",
+	"cad_to_dagmc>=0.9.1",
+	"openmc>=0.15.0",
+	"matplotlib",
+	"netcdf4",
+	"pyyaml",
+      	"pydagmc"
```

Indent inconsistency 
```suggestion
  "pydagmc"
```